groceries_app/views.py

```diff
@@ -1,3 +1,14 @@
-from django.shortcuts import render
+from django.views.generic import TemplateView
 
-# Create your views here.
+
+class HomePageView(TemplateView):
+    template_name = 'home.html'
+
+class AboutPageView(TemplateView):
+    template_name = 'about.html'
+
+class RecipesPageView(TemplateView):
+    template_name = 'recipes/recipes.html'
+
+class PlannerPageView(TemplateView):
```

Comment:
So these are class based views?
What is the naming convention for views? `HomeView` `AboutView` `RecipeView` reads better to me than `HomePageView`